moodlecli/utils.py

```diff
@@ -164,7 +180,7 @@ def setup_duplicate_course(
 
     # Enable enrolment and set enrolment key
     moodle_client.enable_self_enrolment_method(student_enrolment_id)
-    enrolment_key = generate_password()
+    enrolment_key = generate_friendly_password()
```

Comment:
I should have done this when I wrote the test initially, but can we add an assert [here](https://github.com/openstax/raise-moodlecli/blob/098794ea0d383994d93fc92cf41f2c117aa543b3/tests/test_moodle.py#L303) on the value of `utils.CSV_COURSE_ENROLMENT_KEY`? In the test we can use `random.seed` so `generate_friendly_password()` gives a deterministic value.